Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -243,13 +243,22 @@ open class ComposerVC: _ViewController,
         .init()
 
     /// The view controller for selecting image attachments.
-    open private(set) lazy var mediaPickerVC: UIViewController = {
+    open private(set) lazy var mediaPickerVC: UIImagePickerController = {
         let picker = UIImagePickerController()
         picker.mediaTypes = UIImagePickerController.availableMediaTypes(for: .savedPhotosAlbum) ?? ["public.image"]
         picker.sourceType = .savedPhotosAlbum
         picker.delegate = self
         return picker
     }()
```

Comment:
This is also a breaking change. I guess this is a `UIViewController` so that customers can use a different picker, but then it is kinda useless to have a property with `UIViewController` anyway 🤔 But since this is a breaking change, what we can do is create a new property, and maybe call it `imagePicker` and deprecate the `mediaPickerVC`.

Reply:
But it doesn't seem we are using this property any way 🤔 Why is the type changed?

Reply:
Yeah, I have reverted back to keep the same API 👍🏼